code/modules/martial_arts/torque_enhancer.dm

```diff
@@ -0,0 +1,40 @@
+/datum/martial_art/torque
+	name = "Torque enhancer"
+	weight = 500 // You shouldn't be able to override this, it's a passive you actively buy
+	/// What level is the passive at
+	var/level = 1
+
+/datum/martial_art/torque/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	MARTIAL_ARTS_ACT_CHECK
+	var/attack_sound
+	var/list/attack_verb = list()
+	switch(level)
+		if(FLAYER_POWER_LEVEL_ONE)
+			attack_sound = 'sound/weapons/sonic_jackhammer.ogg'
+			attack_verb = list("bashed", "battered")
```

Comment:
I think these attack verbs should be in the present tense to keep with how things are normally presented. "Gray Tider batters you!"